images/egress/router/egress_router_test.go

```diff
@@ -209,6 +209,10 @@ func TestEgressRouterBad(t *testing.T) {
 		}
 		out, err := cmd.CombinedOutput()
 		out_lines := strings.Split(string(out), "\n")
+
+		if len(out_lines) < 2 {
+			continue
```

Comment:
If we're expecting output in order to test it is likely that this should be erroring the test before continue.